site/content/integrations/grafana.md

```diff
@@ -3,19 +3,60 @@ title: "Grafana"
 weight: 3
 ---
 
+## Overview 
 
-M3 supports a variety of Grafana integrations.
+Grafana is an open-source visualization tool used to dashboarding, querying, and monitoring databases, especially used for time series databases. See more [here](https://grafana.com/). 
 
-## Prometheus / Graphite Sources
+M3 supports Grafana integrations for both Prometheus and Graphite/StatsD metrics. It is recommeneded to use Prometheus metrics when integrating with M3DB. 
```

Comment:
```suggestion
M3 supports Grafana integrations for Prometheus and Graphite/StatsD metrics. It is recommeneded to use Prometheus metrics when integrating with M3. 
```

We're still clarifying the difference between M3/M3DB etc, but I think this is correct